bleak/backends/corebluetooth/CentralManagerDelegate.py

```diff
@@ -61,22 +60,128 @@
 DisconnectCallback = Callable[[], None]
 
 
-class CentralManagerDelegate(NSObject):
+class ObjcCentralManagerDelegate(NSObject):
+    """macOS conforming python class for managing the CentralManger for BLE"""
+
+    ___pyobjc_protocols__ = [CBCentralManagerDelegate]
```

Comment:
```suggestion
class ObjcCentralManagerDelegate(NSObject, protocols=[CBCentralManagerDelegate]):
    """
    CoreBluetooth central manager delegate for bridging callbacks to asyncio.
    """
```

Since we are touching this anyway, I'd like to change to the "new" way of specifying the protocol.

Reply:
Even better. The [`protocols=` is the same syntax `rubicon-objc` is using](https://rubicon-objc.readthedocs.io/en/stable/reference/rubicon-objc-api.html#creating-custom-objective-c-classes-and-protocols).